tests/main/interfaces-process-control/task.yaml

```diff
@@ -33,7 +33,7 @@ execute: |
     ps ax | grep -Pq "^ *$pid"
     process-control-consumer.signal "SIGTERM" $pid
     #shellcheck disable=SC2009
-    ! ps ax | grep -Pq "^ *$pid"
+    ps ax | not grep -Pq "^ *$pid"
```

Comment:
Bit on a fence about which version is more readable. This or  `not ( ps ax | grep -Pq "^ *$pid" )`. Feels like `.. | not ..` is not as visible.

Reply:
I think it requires some training to read it this way but I grew to like the fact that “not” is coupled with the command that actually fails.

Reply:
I think it requires some training to read it this way but I grew to like the fact that “not” is coupled with the command that actually fails.